dashboards/google-cloud-composer/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+sample_dashboards:
+  -
+    category: Cloud Composer
+    id: cloud-composer-monitor
+    display_name: Cloud Composer Monitor
+    description: |- 
+      This dashboard provides a one pager overview of any particular Cloud Composer environment to help find bottlenecks related to Environment sizing and performance bottlenecks. 
```

Comment:
1. should Environment be lowercased? 
2. one pager -> one page

Let's model this after existing dashboard descriptions for consistency, e.g.
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-cloud-functions/metadata.yaml
https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/google-cloudsql/metadata.yaml